bin/http-server

```diff
@@ -20,6 +20,7 @@ if (argv.h || argv.help) {
     "  -e --ext           Default file extension if none supplied [none]",
     "  -s --silent        Suppress log messages from output",
     "  --cors             Enable CORS via the 'Access-Control-Allow-Origin' header",
+    "  --corsheaders      Additional CORS headers",
```

Comment:
This doesn't need to be a separate option. We should simply expect `--cors` to also be a string sometimes.